tools/async_to_sync.py

```diff
@@ -564,9 +577,9 @@ def parse_cmdline() -> Namespace:
         "--all", action="store_true", help="process all the files of the project"
     )
     parser.add_argument(
-        "--docker",
-        action="store_true",
-        help=f"run in a docker image with Python {PYVER}",
+        "--container",
```

Comment:
How do you feel about having two mutually exclusive options, `--docker|--podman`?

The reason would be mostly a simpler command line invocation.

If we configure argparse with action="store_const", target="container" the rest of the code would stay largely the same. The command line stripping which would be made simpler, becoming just the likes of `args.remove(f"--{opt.container}")`. The main motivation I have in mind is just command line usage anyway.

Reply:
I'm fine with this approach too; even more if it requires less code.

Reply:
Done now.